### PR TITLE
(chore) Bump playwright

### PR DESCRIPTION
**Pull request by kdaud**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR upgrades playwright to the latest version 1.42.0. See release notes at https://playwright.dev/docs/release-notes

### CODE REVIEW

**Comment by denniskigen** [comment]
Please add a conventional commit label to your PR title. The appropriate one here is `(chore)`. Also, see https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/README.md#updating-playwright.

**Comment by kdaud** [comment]
@denniskigen we've a failing test in program-enrollment. I'm able to reproduce locally and will send a fix soon.

**Comment by kdaud** [comment]
Should I include the fix in this same commit or send a separate PR?

**Comment by denniskigen** [comment]
Within the same PR

**Comment by kdaud** [comment]
@denniskigen I want to do some troubleshooting here in GitHub actions but unfortunately the 'Re-run jobs' functionality is disabled from the user with id `kdaud`

**Comment by kdaud** [comment]
I'm looking into the issue...

**Review by denniskigen — APPROVED** [review]
Thanks, @kdaud. There was a bug in the test where an assertion was checking for the wrong completion date.

File: e2e/specs/program-enrollment.spec.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ test('Add and edit a program enrollment', async ({ page, api }) => {
   await test.step('Then I should see the updated program enrollment in the list', async () => {
     await expect(dataRow).toContainText(/hiv care and treatment/i);
     await expect(dataRow).toContainText(/03-Jul-2023/i);
-    await expect(dataRow).toContainText(/completed on 04-Jul-2023/i);
+    await expect(dataRow).toContainText(/completed on 05-Jul-2023/i);
```
Comment (kdaud) [inline e2e/specs/program-enrollment.spec.ts:74]: @denniskigen the issue is somewhere else. Actually the issue happens when editing the program enrollment.

Comment (kdaud) [reply e2e/specs/program-enrollment.spec.ts:74]: I should be in position to send a commit in the next 10 minutes that fixes the bug.

**Comment by kdaud** [comment]
@denniskigen the latest playwright version runs the E2E tests faster. As a result the modified value for completion date get overridden by the older value causing the test to fail at assertion level. So clearing the date fields before editing the dates resolves the issue.

**Comment by denniskigen** [comment]
Excellent sleuthing work, @kdaud! Faster e2e execution is a great value add!

**Pull request merged**